lib/asciidoctor-pdf/converter.rb

```diff
@@ -1,4 +1,4 @@
-# encoding: UTF-8
+# encoding: utf-8
```

Comment:
I agree. Ruby stores encoding names internally in uppercase, so technically UTF-8 is more correct (though both cases work).